.gitignore

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
same comment as on https://github.com/mt-support/tribe-ext-limit-week-view-time-range/pull/2/ that you shouldn't add a .gitignore to this repo at all.